Sources/Datadog/DDRUMMonitor.swift

```diff
@@ -109,6 +111,19 @@ public class DDRUMMonitor {
         attributes: [AttributeKey: AttributeValue] = [:]
     ) {}
 
+    /// Notifies that the Resource starts being loaded using GET request to given `url`.
```

Comment:
This is not precise - we don't constraint this API to only `GET` if we expect `httpMethod`.